lib/phli.js

```diff
@@ -71,18 +71,19 @@ Phli.prototype.getAddressHistory = function (address, callback) {
   });
 };
 
-Phli.prototype.getPermitInfo = function (permitID, callback) {
+Phli.prototype.getPermitInfo = function (permitID, options, callback) {
   var params = {
     $format: "json"
   };
+  for(option in options) {
```

Comment:
Hmm - is it just because I've named it ```option```? Would simply changing the name address the issue?